keps/sig-storage/20190129-csi-migration.md

```diff
@@ -123,6 +127,57 @@ The detailed design was originally implemented as a [design proposal](https://gi
 
 * All volume operation paths covered by Migration Shim in Beta for >= 1 quarter without significant issues
 
+## Test Plan
+
+### Per-driver migration testing
+
+We will require each plugin/driver provider to set up public CI to run all existing
+in-tree plugin driver tests for their migrated driver.
```

Comment:
I don't understand in the below paragraphs which parts are owned by core k8s and which parts are owned by driver providers. It is clear that we will require providers to display CI results publicly, but which tests does that include? Will any of the tests below run in k8s CI?